.github/workflows/protocol_tests.yml

```diff
@@ -132,19 +132,19 @@ jobs:
 
       - name: Generate migrations and run devchain
         if: success() || failure()
-        run: ./scripts/foundry/create_and_migrate_anvil_l2_devchain.sh
+        run: ANVIL_LOGGING=true ./scripts/foundry/create_and_migrate_anvil_l2_devchain.sh
```

Comment:
Do we need to have it turned on ? My experience was that because of logging it rather slowed down the process (especially when mining 1000s of blocks). We could have it explicitly turned off and if there will be any issue in CI we can turn it rather easily on

Reply:
I can keep it off by default. I remember last discussion with Volpe, where he turned it off 👍 

For reference: https://github.com/celo-org/celo-monorepo/pull/11415#discussion_r2100772197